submissions/seyifunmiOjo/Javascript/arrays/arrays.js

```diff
@@ -0,0 +1,9 @@
+let productList = {
+    productName: "lemons",
+    imageUrl: "images/lemons.jpg", 
+    productPrice: "$5"};
```

Comment:
You didn't fulfil these two criteria of your task 

1. Create an array called `productList`
2. Your `productList` array should contain 3 objects

What you did here is creating a single object not an array of 3 objects